cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/ExecutorConfig.java

```diff
@@ -38,6 +38,21 @@
   public static final boolean DEFAULT_ZOOKEEPER_SECURITY_ENABLED = false;
   public static final String ZOOKEEPER_SECURITY_ENABLED_DOC = "Specify if ZooKeeper is secured, true or false";
 
+  public static final String ZOOKEEPER_SSL_CLIENT_ENABLE_CONFIG = "zookeeper.ssl.client.enable";
+  public static final String ZOOKEEPER_SSL_CNXN_SOCKET_CONFIG = "zookeeper.clientCnxnSocket";
+  public static final String ZOOKEEPER_SSL_KEYSTORE_LOCATION_CONFIG = "zookeeper.ssl.keystore.location";
+  public static final String ZOOKEEPER_SSL_KEYSTORE_PASSWORD_CONFIG = "zookeeper.ssl.keystore.password";
+  public static final String ZOOKEEPER_SSL_KEYSTORE_TYPE_CONFIG = "zookeeper.ssl.keystore.type";
+  public static final String ZOOKEEPER_SSL_TRUSTSTORE_LOCATION_CONFIG = "zookeeper.ssl.truststore.location";
+  public static final String ZOOKEEPER_SSL_TRUSTSTORE_PASSWORD_CONFIG = "zookeeper.ssl.truststore.password";
+  public static final String ZOOKEEPER_SSL_TRUSTSTORE_TYPE_CONFIG = "zookeeper.ssl.truststore.type";
+  public static final String ZOOKEEPER_SSL_PROTOCOL_CONFIG = "zookeeper.ssl.protocol";
+  public static final String ZOOKEEPER_SSL_ENABLED_PROTOCOLS_CONFIG = "zookeeper.ssl.enabled.protocols";
+  public static final String ZOOKEEPER_SSL_CIPHER_SUITES_CONFIG = "zookeeper.ssl.cipher.suites";
+  public static final String ZOOKEEPER_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG = "zookeeper.ssl.endpoint.identification.algorithm";
+  public static final String ZOOKEEPER_SSL_CRL_ENABLE_CONFIG = "zookeeper.ssl.crl.enable";
+  public static final String ZOOKEEPER_SSL_OCSP_ENABLE_CONFIG = "zookeeper.ssl.ocsp.enable";
+
```

Comment:
1. Can we follow the same structure for new configs - i.e. for each config:

```
  /**
   * <code>config.name</code>
   */
  public static final String CONFIG_NAME_CONFIG = "config.name";
  public static final int DEFAULT_CONFIG_NAME = <default-config-value-here>;
  public static final String CONFIG_NAME_DOC = "...";
```

2. Rather than hardcoding config names (e.g. `"zookeeper.ssl.keyStore.location"` and so on), can we refer to relevant config in `KafkaConfig` (e.g. `KafkaConfig.ZkSslKeyStoreLocationProp()` and so on)?

Reply:
3. Also, can we update `Configurations.md` with these new configs?